apps/api/src/app/user/dtos/register-user.dto.ts

```diff
@@ -0,0 +1,26 @@
+import {ApiProperty} from "@nestjs/swagger";
+import {UserRole} from "../models/user-roles";
+export class RegisterUserDto{
+  @ApiProperty({
+    description : 'My name',
+    example : 'John Snow',
+    required: true,
+  })
+  name : string;
+  @ApiProperty({
+    description : 'e-mail where I can be contacted',
+    example : 'JohnSnow@Kingslanding.com',
+    required : true,
+  })
+  email:string;
+  @ApiProperty({
+    description : 'What is my role in all this?',
+    example : 'USER,ADMIN',
+  })
+  roles:UserRole[];
```

Comment:
**issue:** do not allow users to register with any roles they want. This should be handled by the application only (or another admin). In your case set the default to "User"